src/Handler/CurlFactory.php

```diff
@@ -109,9 +109,11 @@ private static function finishError(
     ) {
         // Get error information and release the handle to the factory.
         $ctx = [
-            'errno'  => $easy->errno,
+            'errno' => $easy->errno,
             'error' => curl_error($easy->handle)
-                ?: curl_strerror($easy->errno)
+              ?: (function_exists('curl_strerror')
```

Comment:
You mean remove this line entirely? It will just be:

    'error' => curl_error($easy->handle),